src/app/dashboard/page.tsx

```
@@ -125,63 +118,62 @@ export default function Dashboard() {
             </Link>
           </div>
+
+          {/* Header right */}
           <div className="flex items-center space-x-4">
-            <button
-              onClick={toggleDarkMode}
-              className="text-orange-700 dark:text-orange-400 hover:text-orange-900"
-              aria-label="Toggle Dark Mode"
-            >
-              {isDarkMode ? <Sun className="w-5 h-5" /> : <Moon className="w-5 h-5" />}
+            <button onClick={toggleDarkMode} title="Toggle dark mode">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)